addons/portal/models/mail_message.py

```diff
@@ -93,7 +95,7 @@ def _portal_message_format(self, properties_names, options=None):
         note_id = self.env['ir.model.data']._xmlid_to_res_id('mail.mt_note')
         for message, values in zip(self, vals_list):
             if message_to_attachments:
-                values['attachment_ids'] = message_to_attachments.get(message.id, {})
+                values['attachments'] = message_to_attachments.get(message.id, {})
```

Comment:
`attachment_ids` was better (real python field name) but I don't think it's a good idea to rename in this PR.
Maybe I'll check to make another prep.